doc/bottomSheet.md

```diff
@@ -0,0 +1,135 @@
+# BottomSheet
+
+Mística provides a bottom sheet component that can be used to display a modal-like content from over the main
+content of the screen.
+
+## Basic usage
+
+You can show any content you want inside the bottom sheet by passing it as a child of the component.
+
+```jsx
+import {BottomSheet} from 'mistica';
+
+const MyComponent = () => {
+  const [showSheet, setShowSheet] = useState(false);
+  return (
+    <>
+      <ButtonPrimary onPress={() => setShowSheet(true)}>show bottom sheet</ButtonPrimary>
+      {showSheet && (
+        <BottomSheet onClose={() => setShowSheet(false)}>
+          <Placeholder />
+        </BottomSheet>
+      )}
+    </>
+  );
+};
+```
+
+The sheet will close when the user does the swipe down gesture or when the background overlay is touched. The
+`onClose` callback is called when the closing animation finishes, that's the right place to unmount the sheet
+as shown in the example above.
+
+You can also close the sheet programmatically using the render prop:
+
+```jsx
+import {BottomSheet} from 'mistica';
+
+const MyComponent = () => {
+  const [showSheet, setShowSheet] = useState(false);
+  return (
+    <>
+      <ButtonPrimary onPress={() => setShowSheet(true)}>show bottom sheet</ButtonPrimary>
+      {showSheet && (
+        <BottomSheet onClose={() => setShowSheet(false)}>
+          {({closeModal, modalTitleId}) => (
+            <>
+              <Title1 id={modalTitleId}>My sheet</Title1>
+              <Placeholder />
+              <ButtonPrimary onPress={closeModal}>Close</ButtonPrimary>
+            </>
+          )}
+        </BottomSheet>
+      )}
+    </>
+  );
+};
+```
+
+## Sheet with predefined content
+
+Mística predefines some common bottom sheet patterns for you to use: `RadioListBottomSheet`,
+`ActionsListBottomSheet`, `InfoBottomSheet` and `ActionsBottomSheet`. You can see examples in the storybook.
+
+## `showBottomSheet` imperative api
+
+Instead of using React components, there is an alternative way to show a bottom sheet: using the
+`showBottomSheet` function. For this to work, you need to render a `<BottomSheetRoot/>` somewhere in your app,
+typically where you render the mistica `<ThemeContextProvider/>`, but it could be anywhere.
+
+```jsx
+import {BottomSheetRoot} from '@telefonica/mistica';
+
+export const App = () => {
+  return (
+    <>
+      <BottomSheetRoot />
```

Comment:
I've decided to not include this automatically inside `ThemeContextProvider` to allow webs using mistica to optin to use this. The advantage is that we don't bloat webs with all sheet components JS code if they are not going to use it. Also, it isn't needed to use `BottomSheetRoot` if you don't wan't to use the imperative api